.codeclimate.yml

```diff
@@ -12,6 +12,6 @@ checks:
   similar-code:
     config:
       threshold: 100
-      plugins:
-        git-legal:
-          enabled: true
+plugins:
+    git-legal:
+      enabled: true
```

Comment:
Can we also make sure that we don't allow copyleft? gnu public license and affero?

https://docs.codeclimate.com/docs/git-legal#section-configure-license-policy

Reply:
Of course, I will add that.

Reply:
I am sorry but I don't see these following lines in the PR
```yaml
   config:
      allow_affero_copyleft: false
      allow_strong_copyleft: falsed
```

Reply:
They're there. View the file and you'll see the changes. :)

Reply:
Damn, I was checking only the last commit apparently :'(